format-specs/schema.json

```diff
@@ -11,95 +12,84 @@
+          },
+          "geometry_type": {
+            "oneOf": [
+              {
+                "type": "string",
+                "const": "Unknown"
+              },
+              {
+                "$ref": "#/definitions/geometry_type"
+              },
+              {
+                "type": "array",
+                "items": {
+                  "$ref": "#/definitions/geometry_type"
                 },
-                {
-                  "type": "number",
-                  "description": "The maximum constant latitude line that bounds the rectangle (ymax)."
-                }
-              ]
+                "uniqueItems": true
+              }
+            ],
+            "description": "The geometry type(s) of all geometries, or 'Unknown' if they are not known."
+          },
+          "crs": {
+            "$ref": "https://proj.org/schemas/v0.5/projjson.schema.json"
+          },
+          "edges": {
+            "type": "string",
+            "enum": ["planar", "spherical"],
+            "description": "Name of the coordinate system for the edges. Must be one of 'planar' or 'spherical'. The default value is 'planar'."
+          },
+          "orientation": {
+            "type": "string",
+            "const": "counterclockwise",
+            "description": "Winding order of exterior ring of polygons; interior rings are wound in opposite order. If absent, no assertions are made regarding the winding order."
+          },
+          "bbox": {
+            "type":"array",
+            "description": "Bounding Box of the geometries in the file, formatted according to RFC 7946, section 5.",
+            "items":{
+              "type":"number"
             },
-            "epoch": {
-              "type": "number",
-              "description": "Coordinate epoch in case of a dynamic CRS, expressed as a decimal year."
-            }
+            "oneOf":[
+              {
+                "description": "2D bbox consisting of (xmin, ymin, xmax, ymax)",
+                "minItems":4,
+                "maxItems":4
+              },
+              {
+                "description": "3D bbox consisting of (xmin, ymin, zmin, xmax, ymax, zmax)",
+                "minItems":6,
+                "maxItems":6
+              }
+            ]
           },
-          "additionalProperties": true,
-          "required": ["encoding", "geometry_type"]
+          "epoch": {
+            "type": "number",
+            "description": "Coordinate epoch in case of a dynamic CRS, expressed as a decimal year."
+          }
         }
       }
     }
   },
-  "additionalProperties": true,
-  "required": ["version", "primary_column", "columns"],
-  "$defs": {
+  "definitions": {
     "geometry_type": {
       "type": "string",
-      "enum": ["Point", "LineString", "Polygon", "MultiPoint", "MultiLineString", "MultiPolygon", "GeometryCollection", "Unknown"]
+      "pattern": "^(GeometryCollection|(Multi)?(Point|LineString|Polygon))( Z)?$"
```

Comment:
Added Z and removed Unknown (see above). Could also be a long enum, but thought it might be simpler to use a regexp. We can switch back though if that's preferred. I assume `GeometryCollection Z` is allowed?

Reply:
Do we need to declare `Z` in the geometry type? It might make it a little harder on the writer side

Reply:
That's what the spec states at least... so more a discussion for the general issue tracker?

Reply:
Yes, let's indeed discuss that in a separate issue in case we want to reconsider this

Reply:
See #132

Reply:
This change also removes "Unknown" (which is mentioned [in the text](https://github.com/opengeospatial/geoparquet/blob/811d52bdb57814dbf0c13049481b8f3b2367b31d/format-specs/geoparquet.md#column-metadata) of the spec).

I'll open a separate issue for this, but I wonder if folks would be in support of requiring an array of strings instead of a string or array of strings.  Makes life a bit simpler where this is parsed into a struct and a specific type is useful.

Reply:
As mentioned in my first comment: I moved Unknown (i.e. added as a separate subschema) in the corresponding property.
Reason: It reads as if Unknown is not meant to be used together with other types.

I was also wondering about the array vs string thing and would also say that just always using an array makes things easier (except that it requires a bit more logic to avoid things like ["Point", "Unknown"])

Reply:
> As mentioned in my first comment: I moved Unknown (i.e. added as a separate subschema) in the corresponding property. Reason: It reads as if Unknown is not meant to be used together with other types.

Ah, I should have read more carefully.

> I was also wondering about the array vs string thing and would also say that just always using an array makes things easier (except that it requires a bit more logic to avoid things like ["Point", "Unknown"])

Right, I would use `[]` to signify an unknown geometry type.  See #133.